### PR TITLE
v6.19.0 [skip ci]

### PR DESCRIPTION
**Pull request by hzoo**
https://github.com/babel/babel/blob/changelog/CHANGELOG.md#v6190-2016-11-15

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -12,7 +12,243 @@
...
+```js
+assert(t.isNodesEquivalent(parse("1 + 1"), parse("1+1")) === true);
+```
+
+* `babel-plugin-transform-es2015-modules-systemjs`
+  * [#4789](https://github.com/babel/babel/pull/4789) Support `import()` as contextual import in system module format. ([@guybedford](https://github.com/guybedford))
+
+Support stage-2 `import()` in systemjs.
+
+It doesn't not compile by default; you'll want to add the stage-2 preset or explicitly include `babel-plugin-syntax-dynamic-import`.
```
Comment (DrewML) [inline CHANGELOG.md]: `doesn't not` >> `does not`

File: CHANGELOG.md [hunk 2]
```diff
@@ -12,7 +12,243 @@
...
+Fixes an issue with destructuring parameters being hoisted incorrectly.
+
+Input
+```
+function render({ text }) {
+  return () => (<Component text={text} />);
+}
+```
+
+Ouput
```
Comment (existentialism) [inline CHANGELOG.md]: Nit: Ouput > Output

File: CHANGELOG.md [hunk 3]
```diff
@@ -12,7 +12,243 @@
...
+* Other
+  * [#4802](https://github.com/babel/babel/pull/4802) Add toc [skip ci]. ([@hzoo](https://github.com/hzoo))
+
+#### :house: Internal
+* `babel-plugin-transform-async-to-generator`
+  * [#4837](https://github.com/babel/babel/pull/4837) Fix crlf to lf. ([@lion-man44](https://github.com/lion-man44))
+* Other
+  * [#4807](https://github.com/babel/babel/pull/4807) Chore: FLOW command in makefile and logic in .travis.yml(issue#4710).. ([@sstern6](https://github.com/sstern6))
+
+#### Commiters: 10
```
Comment (existentialism) [inline CHANGELOG.md]: Nit: Commiters > Committers

File: CHANGELOG.md [hunk 4]
```diff
@@ -12,7 +12,243 @@
 _Note: Gaps between patch versions are faulty, broken or test releases._
 
 See [CHANGELOG - 6to5](CHANGELOG-6to5.md) for the pre-4.0.0 version changelog.
+ 
+## v6.19.0 (2016-11-15)
```
Comment (danez) [inline CHANGELOG.md]: 2016-11-16

**Pull request merged**